setup.py

```diff
@@ -3,6 +3,9 @@
 with open('README.rst') as f:
     long_description = f.read()
 
+with open('requirements/dev.txt') as f:
+    required = f.read().splitlines()
```

Comment:
The problem you are solving is understandable, but it is better to solve it differently. It is necessary to make it possible to use aqueduct without installing numpy, since it is only needed for shared memory. Ideally, make it so that you can use shared images without numpy, and install numpy only in cases where you need rummaged numpy